libs/island-ui/core/src/lib/ToggleSwitch/ToggleSwitch.treat.ts

```diff
@@ -0,0 +1,121 @@
+import { style } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+const { spacing, color, typography } = theme
+
+export const toggleSwitchLarge = style({})
+// export const right = style({})
+export const toggleSwitchChecked = style({})
+export const toggleSwitchDisabled = style({})
+export const toggleSwitchHiddenLabel = style({})
+
+export const checkbox = style({})
+
+export const toggleSwitch = style({
+  position: 'relative',
+  display: 'flex',
+  width: 'fit-content',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  marginBottom: spacing[2],
+
+  // link styles
+  color: 'inherit',
+  fontSize: typography.baseFontSize,
+  fontWeight: typography.light,
+  cursor: 'pointer',
+
+  selectors: {
+    '&:hover': {
+      textDecoration: 'none',
+    },
+    '&:focus': {
+      outline: 0,
+    },
+    [`&${toggleSwitchDisabled}`]: {
+      cursor: 'default',
+    },
+  },
+
+  '@media': {
+    print: {
+      display: 'none !important',
+    },
+  },
+})
+
+export const toggleSwitchWide = style({
+  width: '100%',
+})
+
+export const text = style({
+  selectors: {
+    // [`${checked} > &`]: {
+    //   fontWeight: typography.medium,
+    // }
```

Comment:
Unhelpful comment?

Reply:
The plan was to leave that in as example/jumping-off point for possible/likely future modifications – as the designer types start making suggestions for tweaks ... as the whole label thing is somewhat lacking visual specs as of yet.